app/helpers/react_components/student/mentoring_session.rb

```diff
@@ -68,22 +68,13 @@ def num_discussions_with_mentor
       def videos
         return [] if discussion
 
-        # TODO: (required)
         [
           {
-            url: "#",
-            title: "Start mentoring on Exercism..",
-            date: Date.new(2020, 1, 24).iso8601
-          },
-          {
-            url: "#",
-            title: "Best practices writing feedback trrrrrruuuuunnnncaaatteeee",
-            date: Date.new(2020, 1, 24).iso8601
-          },
-          {
-            url: "#",
-            title: "Beginners’ Guide to Mentoring",
-            date: Date.new(2020, 1, 24).iso8601
+            url: Exercism::Routes.doc_path(:using, "feedback/guide-to-being-mentored"),
+            # TODO: Change this
+            thumb: "https://i.vimeocdn.com/video/1230063662?mw=800&mh=450",
```

Comment:
I added this in order for us to be able to configure the thumbnail. I just ripped the thumbnail from the video so we should change this.